docs/1-web-api-server.md

```diff
@@ -0,0 +1,98 @@
+# Web Api server
+
+### Update key-binding
+
+User can update existing key-binding.
+This endpoint is responsible for "Updating key X to value A where key Y has value B".
+
+* HTTP method: **PATCH**
+* Method URL: http://localhost:8080/
+* Accepted content types: `application/json`
+
+#### Request Body: 
+
+* update (required)
+    * must contain one pair key-value e.g. "jira": "new-name2@gmail.com" 
+* where (required)
+    * must contain one pair key-value e.g. "slack": "name1@gmail.com"
+
+```
+{
+    "update": {
+        "jira": "new-name2@gmail.com"
+    },
+    "where": {
+        "slack": "name1@gmail.com"
+    }
+}
+```
+
+#### Response:
+```
+{
+    "result": {
+        "slack": "name1@gmail.com",
+        "jira": "new-name2@gmail.com"
+    },
+    "error": null,
+    "type": "update"
```

Comment:
This `type` field is totally uncessary in the response - user knows what type of request was sent.